### DISCUSSION TITLE
Question about linking MIMIC-CXR subject id and study id to age at time of study

### DISCUSSION THREAD

**Discussion by saahil9jain** [MIT-LCP/mimic-code#1094] [MIMIC-IV-CXR] [answered]
### Prerequisites

* [X ] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description
My goal is to get the age of a patient at the time of a MIMIC-CXR study. Is there a way of doing this using MIMIC-IV or another approach?

The admissions.csv.gz (accessible within https://physionet.org/content/mimiciv/0.4/ in MIMIC-IV Core) has information about age, gender, and race. However, the age is specific to the admission event. I need to link the study (such as s58821758) to the admission event to get the age of the patient at the time of the study. This seems to have been done in the paper CheXclusion (https://arxiv.org/abs/2003.00827) - the author of CheXclusion stated that they had received the data from the MIMIC-CXR team under a limited use agreement - I was wondering if it is possible using MIMIC-IV (or other recently released data from the MIMIC-CXR team).

Thanks!

**Comment by briangow** [comment, marked as answer]
The authors of the CheXclusion paper may have received early access to the MIMIC-CXR dataset.  However, now that MIMIC-CXR is publicly released you should have access to the same information.  Please see this issue [#953](https://github.com/MIT-LCP/mimic-code/issues/953) for information about using the metadata CSV from the [mimic-cxr-jpg](https://physionet.org/content/mimic-cxr-jpg/2.0.0/) project to link to MIMIC-IV core.  The derived table `dicom_metadata_string`, under mimic-cxr on BigQuery provides similar information.  These should provide the necessary information to determine a subjects age at the time a given DICOM was captured.

**Reply by saahil9jain** [reply]
Thanks Brian! I was able to see the StudyDate on the metadata CSV. I then used the anchor age / anchor year in MIMIC-IV core to calculate the age at the time of the study. Let me know if you think there are any issues with that approach.

Appreciate it!

**Reply by briangow** [reply]
This seems like the correct approach.  If it is providing what you were looking for, you can mark my previous response as the answer so other users can quickly find the solution if they have the same question.

**Reply by saahil9jain** [reply]
Done! Thanks Brian.
